.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
And this file... what is this? Is it surplus?

Reply:
https://stackoverflow.com/questions/29435156/what-will-text-auto-eol-lf-in-gitattributes-do

it make sure, if you are on windows, to not change line ending just because you touched a file in editor